pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -4302,6 +4319,702 @@ func Test_server_MultiInsert(t *testing.T) {
 				},
 			}
 		}(),
+		{
+			name: "Boundary Value Testing case 1.1: Success to MultiInsert with 0 value vector (vector type is uint8)",
+			args: args{
+				ctx:  ctx,
+				reqs: genSameReq(100, []float32{0, 0, 0}),
+			},
+			fields: fields{
+				name:              name,
+				ip:                ip,
+				svcCfg:            defaultIntSvcCfg,
+				svcOpts:           defaultSvcOpts,
+				streamConcurrency: 0,
+			},
+			want: want{
+				wantRes: genObjectLocations(100, name, ip),
+			},
+		},
+		{
+			name: "Boundary Value Testing case 1.2: Success to MultiInsert with 0 value vector (vector type is float32)",
+			args: args{
+				ctx:  ctx,
+				reqs: genSameReq(100, []float32{0, 0, 0}),
+			},
+			fields: fields{
+				name:              name,
+				ip:                ip,
+				svcCfg:            defaultF32SvcCfg,
+				svcOpts:           defaultSvcOpts,
+				streamConcurrency: 0,
+			},
+			want: want{
+				wantRes: genObjectLocations(100, name, ip),
+			},
+		},
+		{
+			name: "Boundary Value Testing case 2.1: Success to MultiInsert with min value vector (vector type is uint8)",
+			args: args{
+				ctx:  ctx,
+				reqs: genSameReq(100, []float32{math.MinInt, math.MinInt, math.MinInt}),
+			},
+			fields: fields{
+				name:              name,
+				ip:                ip,
+				svcCfg:            defaultIntSvcCfg,
+				svcOpts:           defaultSvcOpts,
+				streamConcurrency: 0,
+			},
+			want: want{
+				wantRes: genObjectLocations(100, name, ip),
+			},
+		},
+		{
+			name: "Boundary Value Testing case 2.2: Success to MultiInsert with min value vector (vector type is float32)",
+			args: args{
+				ctx:  ctx,
+				reqs: genSameReq(100, []float32{math.SmallestNonzeroFloat32, math.SmallestNonzeroFloat32, math.SmallestNonzeroFloat32}),
```

Comment:
why did you use `SmallestNonzeroFloat32`?
the value of the vector is not always positive.

Reply:
thank you for your comment.
Since this value is copied from insert test, I also fixed the insert test